src/plugins/timelist/TimelistComponent.vue

```diff
@@ -38,39 +38,41 @@
       >
       </expanded-view-item>
     </template>
-    <div v-else class="c-table c-table--sortable c-list-view c-list-view--sticky-header sticky">
-      <table class="c-table__body js-table__body">
-        <thead class="c-table__header">
-          <tr>
-            <list-header
-              v-for="headerItem in headerItems"
-              :key="headerItem.property"
-              :direction="
-                defaultSort.property === headerItem.property
-                  ? defaultSort.defaultDirection
-                  : headerItem.defaultDirection
-              "
-              :is-sortable="headerItem.isSortable"
-              :aria-label="headerItem.name"
-              :title="headerItem.name"
-              :property="headerItem.property"
-              :current-sort="defaultSort.property"
-              @sort="sort"
+    <template v-else>
+      <div class="c-table c-table--sortable c-list-view c-list-view--sticky-header sticky">
+        <table class="c-table__body js-table__body">
+          <thead class="c-table__header">
+            <tr>
+              <list-header
+                v-for="headerItem in headerItems"
+                :key="headerItem.property"
+                :direction="
```

Comment:
I know this was in here from before, but can we move this logic out of the template? Maybe we make headerItems a computed, something like:

```js
headerItems() {
    return headerItems.map(item => ({
      ...item,
      direction: this.defaultSort.property === item.property
        ? this.defaultSort.defaultDirection
        : item.defaultDirection
    }));
  }

Reply:
I like it!